frontend/src/scenes/settings/user/PersonalAPIKeys.tsx

```diff
@@ -85,18 +238,38 @@ function PersonalAPIKeysTable(): JSX.Element {
                     title: 'Label',
                     dataIndex: 'label',
                     key: 'label',
+                    render: function RenderLabel(label, key) {
+                        return (
+                            <Link
+                                subtle
+                                className="text-left font-semibold truncate"
+                                onClick={() => setEditingKeyId(key.id)}
+                            >
```

Comment:
Not sure I agree with the label thing. We do this elsewhere as well (annotations for example).

Happy to additionally add the drop down though

Reply:
The pattern is that those titles are links, which is the case in annotations. But most importantly the title is not the only way to initiate editing (because it's not obvious), e.g. annotations also have an "Edit" button.